### PR TITLE
DOC-1803: Create `admon-requires-6.3v.adoc` partial for future release

### PR DESCRIPTION
**Pull request by MitchCTiny**
Related Ticket: DOC-1803

Description of Changes:
* Added `admon-requires-6.3v.adoc` as partial for future release

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] Files has been included where required (if applicable)
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Comment by brian-forte** [comment]
This may be superstition on my part, but, over the past 30+ years I’ve had various systems tell me off for giving them text files without a new-line character as the last character in the file.

Literally thousands of *No newline at end of file* messages and warning (and the multiple variations thereof) has me almost reflexively inclined to add a hard-return to even one-line files.

But, this is a learned, almost Pavlovian response, with only a long history of wanting to avoid warnings in log files and at STD OUT as evidence for it being the right thing to do.

Hence this comment, rather than the edit I really want to make (for, perhaps, no good reason).

**Comment by ephox-mogran** [comment]
@brian-forte the previous files also had a trailing newilne, so you can argue on the basis of consistency, as well. I know in some of our scripts for other projects we've been burnt by not having trailing newlines, because the last line of a list wasn't processed.

**Review by ephox-mogran — APPROVED** [review]

File: modules/ROOT/partials/misc/admon-requires-6.3v.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+NOTE: This feature is only available for {productname} 6.3 and later.
```
Comment (brian-forte) [inline modules/ROOT/partials/misc/admon-requires-6.3v.adoc:1]:
```suggestion
NOTE: This feature is only available for {productname} 6.3 and later.

```
Adding a blank line.

Consequent to @ephox-mogran’s [comment](https://github.com/tinymce/tinymce-docs/pull/2519#issuecomment-1254372895), I’m taking the leap from [commentary](https://github.com/tinymce/tinymce-docs/pull/2519#issuecomment-1253564668) to concrete suggestion.

**Review by tiny-ben-tran — APPROVED** [review]
Don't think this is scalable. Why can't we parameterised the version number?

**Comment by MitchCTiny** [comment]
> Don't think this is scalable. Why can't we parameterised the version number?

Something to think about, I'm happy to replace all the current version partials with a new parameter and a single partial.

**Comment by lnewson** [comment]
This **cannot** use variables as this needs to be hardcoded. Now we could have something where you need to specify variables to include this, but that is very messy. This is generally the least intrusive options and it's fairly scalable as you only need one per minor release.

**Comment by brian-forte** [comment]
> > Don't think this is scalable. Why can't we parameterised the version number?
> 
> Something to think about, I'm happy to replace all the current version partials with a new parameter and a single partial.

A further something to think about: a single parameter would make some statements technically untrue.

Something that requires 6.1 will, with a single parameter, be presented as requiring 6.3 when 6.3 comes out.

**Comment by MitchCTiny** [comment]
> > > Don't think this is scalable. Why can't we parameterised the version number?
> > 
> > 
> > Something to think about, I'm happy to replace all the current version partials with a new parameter and a single partial.
> 
> A further something to think about: a single parameter would make some statements technically untrue.
> 
> Something that requires 6.1 will, with a single parameter, be presented as requiring 6.3 when 6.3 comes out.

I meant a single parameter which is declared immediately before including a partial, then likely switched back to 6.0 after the partial, and by default. But I agree with above, this method is simple and won't cause confusion.

**Pull request merged**